lib/manager/poetry/artifacts.js

```diff
@@ -18,11 +18,17 @@ async function getArtifacts(
     logger.debug('No updated poetry deps - returning null');
     return null;
   }
-  const lockFileName = 'poetry.lock';
+  const splitPackageFileName = packageFileName.split('/');
```

Comment:
Better to use `path` functions here. Check out some other artifacts functions for examples